setup/README.md

```diff
@@ -56,81 +56,76 @@ Before you can finally set up Connaisseur, you will need to configure its connec
 
 #### Configure Notary
 
-In `helm/values.yaml`, we first need to specify the location of our notary under `notary.host`:
+In `helm/values.yaml`, we first need to specify the location of the to be used notary instance. Connaisseur supports the configuration of multiple notary instances, should your image signatures be at different locations. Per default the official DockerHub notary is configured, with the public key for all official images, but for now let's assume you want to add a new notary instance. Simply add a new entry in the `notary` field, specifying a `name`, `host` and a list of public root keys (`rootKeys`). The `name` should be unique among all notary configurations, the `host` points to the host address of the notary instance and the list of public root keys will be used to verify the image signatures residing in this notary instance. Why a list, you may ask? Just take a look at the the official DockerHub notary, where the official images are signed with a different key as your own images you can push to DockerHub. If you want to verify both, official images and your own (all residing on DockerHub), you'll need multiple keys. How to obtain these will be described further below ([here](#retrieving-the-public-root-key)).
 
-```yaml
-notary:
-  # domain to the notary server. can be `null` or non-existant to use
-  # the public Docker Hub notary server
-  host: notary.docker.io
-```
-
-If you use a public Docker Hub repository, you can keep `notary.auth.enabled` at `false`, skip the following and directly go to the next step [configuring your public key](#set-your-public-key-as-trust-anchor). In case you use a notary server that requires authorization or decide to use a private repository on Docker Hub, you need to pass your credentials to Connaisseur. For simplicity in a test environment, you can just set `notary.auth.enabled` to `true`, `notary.auth.user` to your username or Docker Hub ID and `notary.auth.password` to your password:
+In addition to these three mandatory settings, a self-signed certificate and authentication can be added, as well as marking a notary belonging to a azure container registry (acr). The self-signed certificate can be added with the `selfsignedCert` field, where it will be pasted in a PEM format. The authentication detail can be defined in the `auth` field, where either the credentials are given directly with `auth.user` and `auth.password` or a predefined [Kubernetes secret](https://kubernetes.io/docs/concepts/configuration/secret/) is referenced in the `auth.secretName` field. The data of the predefined secret needs to be a `cred.yaml` with the fields `USER` and `PASS`. Lastly the `isAcr` can be used to mark the notary as being part of an azure container registry, as these types of notaries present a slightly different kind of behavior. All in all a sample notary configuration might look like this:
```

Comment:
```suggestion
In addition to these three mandatory settings, a self-signed certificate and authentication can be added, as well as marking a notary belonging to an Azure Container Registry (ACR). The self-signed certificate can be added with the `selfsignedCert` field, where it will be pasted in PEM format. The authentication detail can be defined in the `auth` field, where either the credentials are given directly with `auth.user` and `auth.password` or a predefined [Kubernetes secret](https://kubernetes.io/docs/concepts/configuration/secret/) is referenced in the `auth.secretName` field. The data of the predefined secret needs to be a `cred.yaml` with the fields `USER` and `PASS`. Lastly the `isAcr` can be used to mark the notary as being part of an Azure Container Registry, as these types of notaries present a slightly different kind of behavior. All in all a sample notary configuration might look like this:
```